RustLeanModels/Basic.lean

```diff
@@ -52,44 +55,61 @@ theorem Eq_iff_of_unique_and_mp : unique p → (∀ y, (x = y) → p y) → (∀
   unfold unique at g1; exact g1 x y g2 g3
 
 @[simp]
-lemma not_sym :  ¬ x = y →  ¬ y = x := by intro g; by_contra; rename_i g1; symm at g1; contradiction
+theorem not_sym :  ¬ x = y →  ¬ y = x := by intro g; intro h; have h := Eq.symm h; contradiction
```

Comment:
Could combine the two `intro's (and conclude with simp_all?)
```
theorem not_sym : ¬ x = y →  ¬ y = x := by intro g h; simp_all
```

Reply:
Done